MPF.Core/UI/ViewModels/CreateIRDViewModel.cs

```diff
@@ -0,0 +1,1071 @@
+﻿#if NET6_0_OR_GREATER
+using LibIRD;
```

Comment:
If creating IRDs is limited to modern .NET anyway, you can scope the entire file here inside of the `#if NET6_0_OR_GREATER`

Reply:
You would think so, but it breaks the .xaml as you cannot have preprocessor conditionals inside XAML
Unless you can figure out how to exclude assemblies.

Reply:
This is the view model, not the XAML. The button in the menu to access this could just be hidden or only enabled for modern dotnet.

Reply:
The XAML still complains about the non-existence of the view model, and if I define a dummy view model it complains that the methods are not defined. If there's an easy way to do this I'm not finding it.

Reply:
That is incredibly frustrating. Okay, we can keep it this way until conditional compilation can be figured out.